### PR TITLE
Remove redundant log message

### PR DESCRIPTION
**Pull request by deldesir**

### CODE REVIEW

**Comment by holta** [comment]
Related:

- PR #148

**Comment by holta** [comment]
Explain why redundant, with screenshot or example?

**Comment by deldesir** [comment]
This log message is prone to too much repetition. Logging with the name of the shelf is enough. No need to repeat message of every shelf with similar name found like:

```
A public shelf with the name 'Test' already exists
A public shelf with the name 'Test 1' already exists
A public shelf with the name 'Test 2' already exists
A public shelf with the name 'Test 3' already exists
A public shelf with the name 'Test 4' already exists
Shelf 'Test 5' added
```

File: cps/shelf.py [hunk 1]
```diff
@@ -403,7 +403,6 @@ def check_shelf_is_unique(title, is_public, shelf_id=False):
                                    .first() is None
 
         if not is_shelf_name_unique:
-            log.error("A public shelf with the name '{}' already exists.".format(title))
```
Comment (holta) [inline cps/shelf.py:406]: Any change the line should be commented out and clarified, e.g. as follows?

```
# log.info("A public shelf with similar name '{}' already exists.".format(title))

Comment (deldesir) [reply cps/shelf.py:406]: Ok

Comment (deldesir) [reply cps/shelf.py:406]: Done

File: cps/shelf.py [hunk 2]
```diff
@@ -403,7 +403,7 @@ def check_shelf_is_unique(title, is_public, shelf_id=False):
...
+            # log.error("A public shelf with the name '{}' already exists.".format(title))
```
Comment (holta) [inline cps/shelf.py:406]:
```suggestion
            # log.info("A public shelf with similar name '{}' already exists.".format(title))
```

Comment (holta) [reply cps/shelf.py:406]: Ok?

**Pull request merged**